store_test.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAddToStore(t *testing.T) {
+	store := NewToDoStore()
+	for i := 1; i <= 10; i++ {
+		// each task gets id + 1 of previous task id
+		task := new(Task)
```

Comment:
I don't frequently see `new` used to create a zero-value instance of a struct in go. There's nothing wrong with this, but I think its slightly more idiomatic to use `Task{}` for this instead.